### PR TITLE
[WDFT-REMOTE-2007] Alejandro Hernández

### PR DESCRIPTION
**Pull request by alejandrohj**
Hi TA,

It's almost done, rest the split and reset iterations, easy, but I have to finish it tonight.

### CODE REVIEW

**Comment by alejandrohj** [comment]
It's done.

File: javascript/chronometer.js [hunk 1]
```diff
@@ -1,26 +1,43 @@
...
+    return Math.floor(this.currentTime - this.getMinutes()*60) 
+  }
+  getMiliSecons(){
+    if(this.miliseconds >= 100){this.miliseconds = 0}
+    return this.miliseconds;
   }
-  twoDigitsNumber() {
-    // ... your code goes here
+  twoDigitsNumber(num) {
+    return num.toString().padStart(2,0);
```
Comment (IH-WebDev-TA-Remote) [inline javascript/chronometer.js:29]: Excellent use of padStart, it is kinda of an unknown method that works perfectly here.

File: javascript/chronometer.js [hunk 2]
```diff
@@ -1,26 +1,43 @@
...
   getMinutes() {
-    // ... your code goes here
+    return Math.floor(this.currentTime/60);
   }
   getSeconds() {
-    // ... your code goes here
+    return Math.floor(this.currentTime - this.getMinutes()*60) 
+  }
+  getMiliSecons(){
+    if(this.miliseconds >= 100){this.miliseconds = 0}
```
Comment (IH-WebDev-TA-Remote) [inline javascript/chronometer.js:25]: Little tip, when doing if conditionals in one line you can remove the curly brackets. Looks better and still easy to read.

File: javascript/index.js [hunk 3]
```diff
@@ -14,52 +16,91 @@ let milUni = document.getElementById('milUni');
...
 
 function printTime() {
-  // ... your code goes here
+  printSeconds();
+  printMinutes();
 }
 
 function printMinutes() {
-  // ... your code goes here
+  minUni.innerHTML = (chronometer.getMinutes())%10;
```
Comment (IH-WebDev-TA-Remote) [inline javascript/index.js:24]: You could only run the function once to get the currentTime in a variable and then apply the %10 or /10 for each dom element. It is an additional line but only calling the function once would be better for performance.

File: javascript/index.js [hunk 4]
```diff
@@ -14,52 +16,91 @@ let milUni = document.getElementById('milUni');
...
 function printMilliseconds() {
-  // ... your code goes here
-}
-
-function printSplit() {
-  // ... your code goes here
+  milUni.innerHTML = (chronometer.getMiliSecons())%10;
+  milDec.innerHTML = Math.floor((chronometer.getMiliSecons())/10)
 }
+//Are these necesary?
```
Comment (IH-WebDev-TA-Remote) [inline javascript/index.js:38]: Not necessary but they work for having a more modular code. You could put the dom lines that add the clear everything or add the split time to the dom here and just call this functions in the event listeners.

File: javascript/index.js [hunk 5]
```diff
@@ -1,9 +1,11 @@
 const chronometer = new Chronometer();
-
 // get the buttons:
+
 const btnLeft = document.getElementById('btnLeft');
 const btnRight = document.getElementById('btnRight');
-
+let status= {
```
Comment (IH-WebDev-TA-Remote) [inline javascript/index.js:6]: Amazing idea creating this boolean, this is React logic! ;)

**Comment by IH-WebDev-TA-Remote** [comment]
Great work, I had to be very picky to find things to improve in the code. You are doing amazing, keep it up!

**Pull request closed without merging**